HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 History
 =======
 
+* Make ``re_path`` -> ``path`` fixer also convert ``include()``'s with unterminated regexes.
```

Comment:
reStructuredText requires an escape if putting something right next to inline syntax

looks like GitHub’s renderer is fine with it but PyPI’s won't be
```suggestion
* Make ``re_path`` -> ``path`` fixer also convert ``include()``\'s with unterminated regexes.
```